services/docker-compose.yml

```diff
@@ -34,11 +34,36 @@ services:
     networks:
       - default
     environment:
+      - AUTOSCALING_MONITORED_NODES_LABELS=${AUTOSCALING_MONITORED_NODES_LABELS}
+      - AUTOSCALING_MONITORED_SERVICES_IMAGE_NAMES=${AUTOSCALING_MONITORED_SERVICES_IMAGE_NAMES}
+      - AUTOSCALING_MONITORED_SERVICES_LABELS=${AUTOSCALING_MONITORED_SERVICES_LABELS}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - AWS_ALLOWED_EC2_INSTANCE_TYPE_NAMES=${AWS_ALLOWED_EC2_INSTANCE_TYPE_NAMES}
+      - AWS_AMI_ID=${AWS_AMI_ID}
+      - AWS_DNS=${AWS_DNS}
+      - AWS_ENDPOINT=${AWS_ENDPOINT}
+      - AWS_KEY_NAME=${AWS_KEY_NAME}
+      - AWS_REGION_NAME=${AWS_REGION_NAME}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+      - AWS_SECURITY_GROUP_IDS=${AWS_SECURITY_GROUP_IDS}
+      - AWS_SUBNET_ID=${AWS_SUBNET_ID}
       - LOG_LEVEL=${LOG_LEVEL:-WARNING}
```

Comment:
so I guess it is the LOG_LEVEL you're talking about here. I think WARNING is a sensible default. I would not make the the service fails because the log level is not set.